src/AltinnCore/Common/Services/Implementation/RepositorySI.cs

```diff
@@ -186,9 +187,9 @@ public void CreateApplication(string org, string appName)
                 LastChangedDateTime = DateTime.UtcNow,
                 LastChangedBy = developer
             };
-
+            
             appMetadata.Title = new Dictionary<string, string>();
-            appMetadata.Title.Add("nb", appName);
+            appMetadata.Title.Add("nb", appTitle);
```

Comment:
should add a check if the string is empty. 
In appMetadataController you call this method with an empty string and the dictionary will store an entry that doesn't provide any information.

Reply:
Added null check.